src/components/screens/Validacion/components/Text/index.module.scss

```
@@ -0,0 +1,26 @@
+.container{
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 30px;
+}
+
+  .texto {
+    width: 384px;
+    color: #525256;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 24px;
+    font-weight: bold;
+    letter-spacing: 0;
+  }
+
+  .revision {
+    width: 92px;
+    color: #7758A2;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 24px;
+    font-weight: bold;
+    letter-spacing: 0;
+    margin-bottom: 36px;
+    margin-top: 14px;
+  }
```

Cuidado con la identación:

```suggestion
.texto {
  width: 384px;
  color: #525256;
  font-family: 'Montserrat', sans-serif;
  font-size: 24px;
  font-weight: bold;
  letter-spacing: 0;
}

.revision {
  width: 92px;
  color: #7758A2;
  font-family: 'Montserrat', sans-serif;
  font-size: 24px;
  font-weight: bold;
  letter-spacing: 0;
  margin-bottom: 36px;
  margin-top: 14px;
}
```